docs/image.md

```diff
@@ -245,7 +245,7 @@ blurRadius: the blur radius of the blur filter added to the image.
 | ------ |
 | number |
 
-> Tip : IOS you will need to increase `blurRadius` more than `5`
+> Tip : On IOS, you will need to increase `blurRadius` by more than `5`
```

Comment:
```suggestion
> Tip: On IOS, you will need to increase `blurRadius` by more than `5`.
```

Reply:
Okay, thanks for the suggestion.

Reply:
Thank you for the quick update, let's ship it! 🙂